bluecellulab/cell/template.py

```diff
@@ -76,14 +76,19 @@ def get_cell(
             cell = getattr(neuron.h, self.template_name)(
                 gid, morph_dir, morph_fname
             )
-        elif template_format == "v6_ais_scaler":
+        elif template_format == "v6_adapted":
```

Comment:
What if some old hoc files are still using the `v6_ais_scaler` in their template? Shouldn't we support them anymore?

Reply:
E.g. this one

https://raw.githubusercontent.com/BlueBrain/BlueCelluLab/main/examples/2-sonata-network/circuit_sonata_quick_scx_multi_circuit/components/CircuitA/hoc/cADpyr_L2TPC.hoc

Reply:
this is not used much, only by me once for NCx project which got discontinued. I will try to update the tests/examples as much as I can

Reply:
I see. That's good news. 

One thing that worries me however is that some cells in Neurodamus' Zenodo link also use AIS_scaler.

Zenodo: https://zenodo.org/record/8075202

Cell1: https://github.com/BlueBrain/neurodamus/blob/826e2511684d3b7c96c315f93492bc3e2603d268/tests/simulations/usecase3/CircuitA/hoc/cNAC_L23BTC.hoc#L22
Cell2: https://github.com/BlueBrain/neurodamus/blob/826e2511684d3b7c96c315f93492bc3e2603d268/tests/simulations/usecase3/CircuitA/hoc/cADpyr_L2TPC.hoc#L22

Those cells are most likely extracted from a SONATA circuit.
The Part II: Physiology and Experimentation paper says:
> 7. Open
Source: A seven column subvolume of the model is available on Zenodo (see data availability
statement).

https://www.biorxiv.org/content/10.1101/2023.05.17.541168v1.full.pdf
We must make sure that seven column subvolume does not contain the AIS_scaler hocs. Otherwise users cannot load those cells.

Reply:
ah these is my stuff haha: These files are from the thal-ncx model.
Morphologies come from:

/gpfs/bbp.cscs.ch/project/proj82/home/gevaert/morphology_release/mouse-scaled/scaled_output/09_CloneMorphologies-asc/

HOC files are from:

/gpfs/bbp.cscs.ch/project/proj82/singlecell/emodel_release/complete_emodel_release_new_try/hoc_files/

this is thal-ncx things, not sscx related to these papers, I'm not sure why this is there tbh

Reply:
but I will also need to update neurodamus I think, I need to check